### DISCUSSION TITLE
On upload trigger

### DISCUSSION THREAD

**Discussion by migue802** [Natizyskunk/vscode-sftp#109] [Q&A] [answered]
Is there any option to execute a command when SFTP finishes uploading??

**Comment by Natizyskunk** [comment, marked as answer]
You should be able to create your own command in vscode that'll listen to any other command and do something based on it.

**Reply by migue802** [reply]
Hmm... yeah that's true. But I was asking if the extension itself haves a function like that.

**Reply by Natizyskunk** [reply]
Not as i remember, no.
